iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ValidationUtil.java

```diff
@@ -51,17 +51,6 @@ public static void checkActivePowerLimits(Validable validable, double minP, doub
         }
     }
 
-    public static void checkActivePowerLimits(Validable validable, double minP, double maxP, double p) {
-        checkActivePowerLimits(validable, minP, maxP);
-
-        if (p > maxP) {
-            throw new ValidationException(validable, "invalid active power p > maxP: " + p + " > " + maxP);
-        }
-        if (p < minP) {
-            throw new ValidationException(validable, "invalid active power p < minP: " + p + " < " + minP);
-        }
-    }
```

Comment:
I think you can keep this method as `@Deprecated(since="4.3.0")`, I'm not sure it is not used in rte-core for example... (in extensions)